draft-ietf-scitt-architecture.md

```diff
@@ -367,43 +367,51 @@ This section describes at a high level, the three main roles and associated proc
 
 ## Transparency Service
 
-An important function of a Transparency Service is to maintain Registration Policies for the Append-only Log.
-The Append-only Log is the verifiable data structure which registers Signed Statements and supports the production of Receipts.
+Transparency Services MUST feature an Append-only Log. The Append-only Log is the verifiable data structure which registers Signed Statements and supports the production of Receipts.
 
 All Transparency Services MUST expose APIs for the registration of Signed Statements and issuance of Receipts.
 
 Transparency Services MAY support additional APIs for auditing, for instance, to query the history of Signed Statements.
 
-Typically a Transparency Services has a single Issuer identity which is present in the `iss` claim of Receipts for that service.
+Typically a Transparency Service has a single Issuer identity which is present in the `iss` claim of Receipts for that service.
 
 Multi-tenant support can be enabled through the use of identifiers in the `iss` claim, for example, `ts.example` may have a distinct Issuer identity for each sub domain, such as `customer1.ts.example` and `customer2.ts.example`.
 
-### Initialization {#ts-initialization}
+### Registration Policies
 
-The Append-only Log is empty when the Transparency Service is initialized.
-The first entry that is added to the Append-only Log MUST be a Signed Statement including key material.
-The second set of entries are Signed Statements for additional domain-specific Registration Policy.
-The third set of entries are Signed Statements for Artifacts.
-From here on a Transparency Service can check Signed Statements on registration via policy (that is at minimum, key material and typically a Registration Policy) and is therefore in a reliable state to register Signed Statements about Artifacts or a new Registration Policy.
+Registration Policies refer to additional checks over and above the Mandatory Registration Checks that are performed before a Signed Statement is accepted to be registered to the Append-only Log.
 
-### Registration Policies
+Transparency Services MUST maintain Registration Policies which govern whether or not a given Signed Statement is eligible for registration.
+
+Registration Policies MUST be made transparent and available to all clients of the Transparency Service by registering them as Signed Statements on the Append-only Log and providing a means of fetching and verifying the corresponding Transparent Statement.
+
+Transparency Services MUST reject any attempt to register Signed Statements until at least one Registration Policy is registered.
+
```

Comment:
This is a deadlock... can't register a policy without a policy.

Reply:
A registration policy is just another signed statement, so how do we release the deadlock?

Reply:
I just noticed this [initialization proposal below](https://github.com/ietf-wg-scitt/draft-ietf-scitt-architecture/pull/201/files#diff-585fb424519b79cf000445f3425fa56d328cbaca5d2a622740ddc4e5ca91dbe1R412): 

> Since a Registration Policy is required prior to the registration of any Signed Statements, a means is required to configure the first Registration Policy that is not the standard issuance of a Signed Statement.
> Transparency Services MUST support at least one of these methods:

Reply:
I propose to resolve this comment given the presence of the Initialization section unless @OR13 still objects